packages/app-builder-lib/scheme.json

```diff
@@ -6416,6 +6416,11 @@
       ],
       "description": "macOS DMG options."
     },
+    "downloadAlternateFFmpeg": {
```

Comment:
For the schema, we'll also need to run `pnpm generate-all` to update the docs+schema together

Reply:
aha, ok, thanks for the tip. I'll run that and submit another commit with the updates.